src/libraries/Native/Unix/System.Globalization.Native/pal_calendarData.c

```diff
@@ -417,12 +416,12 @@ static int32_t EnumAbbrevEraNames(const char* locale,
     // The C-API for ICU provides no way to get at the abbreviated era names for a calendar (so we can't use EnumSymbols
     // here). Instead we will try to walk the ICU resource tables directly and fall back to regular era names if can't
     // find good data.
-    char localeNameBuf[ULOC_FULLNAME_CAPACITY];
-    char parentNameBuf[ULOC_FULLNAME_CAPACITY];
+    char localeNameBuf[ULOC_FULLNAME_CAPACITY]{};
```

Comment:
Why is it better to add superfluous `{}` ?

Reply:
It's declared but not initialized, so with `{}` it will get initialized to default value rather than holding a garbage value.

Reply:
This zero initialization of buffers is not free. We do not want to pay for it.

Reply:
Oh ok. I will get this reverted with the next commit!